tests/test_imperva.py

```diff
@@ -28,7 +28,7 @@ def test_imperva_incapsula(record_property, setup_wordlist, setup_splunk, setup_
 
     sendsingle(message, setup_sc4s[0], setup_sc4s[1][514])
 
-    st = env.from_string("search _time={{ epoch }} index=netwaf host=\"{{ host }}\" sourcetype=\"cef\" source=\"Imperva:Incapsula\"")
+    st = env.from_string("search _time={{ epoch }} index=netwaf host=\"{{ host }}\" sourcetype=\"imperva:incapsula\" source=\"Imperva:Incapsula\"")
```

Comment:
Revert this change the sourcetype is cef on this one

Reply:
Reverted...